core/modules/simpletest/tests/form_test.module

```diff
@@ -1789,14 +1789,14 @@ function _form_test_disabled_elements($form, &$form_state) {
     '#title' => 'date',
     '#disabled' => TRUE,
     '#default_value' => array(
-      'day' => 19,
-      'month' => 11,
-      'year' => 1978,
+      'day' => 16,
+      'month' => 01,
```

Comment:
Ah sorry for my previous comment, the actual problem is the number with starting "0"...

Numbers starting with 0 is considered octal values, either wrap it in "" or remove the leading zero. I'd suggest to use "".

Reply:
Good catch @indigoxela 👍 ...why use `'` though? (which converts the number to string AFAIK)

Reply:
> why use ' though?

If values are provided via form elements, they're usually strings (in `$form_state['values']`).

Reply:
Thanks @indigoxela! Fixed.